src/app/app.component.ts

```diff
@@ -21,13 +22,18 @@ import BreadcrumbsComponent from '@components/breadcrumbs/breadcrumbs.component'
     schemas: [CUSTOM_ELEMENTS_SCHEMA],
 
 })
-export class AppComponent {
-  constructor(private router: Router) {}
-
+export class AppComponent implements OnInit {
+  constructor(private router: Router,private userStoreService: UserStoreService) {}
+  
+  
   title = 'Angular Demo';
   mobileNavOpen = false;
   mode: 'tds-mode-light' | 'tds-mode-dark' = 'tds-mode-light';
   modeVariant:'tds-mode-variant-primary' | 'tds-mode-variant-secondary' = 'tds-mode-variant-primary';
+  userName = '';
+  placeOfWork = '';
+  notifications: {}[] = [];
```

Comment:
if the BehaivourSubject is typed properly this should change to `notifications: Notification[] = [];`